hazelcast/src/main/java/com/hazelcast/internal/serialization/impl/DefaultSerializationServiceBuilder.java

```diff
@@ -235,7 +235,8 @@ public SerializationService build() {
                     ((HazelcastInstanceAware) serializer).setHazelcastInstance(hazelcastInstance);
                 }
 
-                ((AbstractSerializationService) ss).registerGlobal(serializer);
+                ((AbstractSerializationService) ss)
```

Comment:
I would pull out a local variable.

AbstractSerializationService ss = ....
ss.registerGlobal(...)

Else the code is so hard to digest.